eventhubs/data-plane/azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/impl/PartitionReceiverImpl.java

```diff
@@ -279,6 +279,6 @@ public String getPartitionId() {
 
         return this.receiverOptions != null && this.receiverOptions.getReceiverRuntimeMetricEnabled()
                 ? new Symbol[]{AmqpConstants.ENABLE_RECEIVER_RUNTIME_METRIC_NAME}
-                : null;
+                : new Symbol[0];
```

Comment:
This changes behavior and shouldn't be done.

Reply:
so don't change the return null case and suppression this error?

Reply:
Correct.